spyder/plugins/ipythonconsole.py

```diff
@@ -1359,7 +1367,10 @@ def _new_connection_file(self):
         """
         # Check if jupyter_runtime_dir exists (Spyder addition)
         if not osp.isdir(jupyter_runtime_dir()):
-            os.makedirs(jupyter_runtime_dir())
+            try:
+                os.makedirs(jupyter_runtime_dir())
+            except PermissionError as pe:

```

Comment:
Last thing: there's no need of capturing the error (`as pe`) because we're not doing anything with it :-)